### PR TITLE
quickstart instructions in readme

### PR DESCRIPTION
**Pull request by JonathanMH**
Thanks for the awesome post on it, I just found it on HN!

Had a bit of trouble figuring out the process of trying it out, am I doing it right? :D

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
 This is a toy raytracer I wrote in rust to learn how raytracers work. I also
 wrote a series of posts on it starting
 [here](https://bheisler.github.io/post/writing-raytracer-in-rust/).
+
+To run the example, clone the repository and
+
+```
+cd raytracer
+cd app
+cargo run scenes/test.json scenes/checkerboard.png
```
Comment (bheisler) [inline README.md:10]: The second argument is the output file. This will overwrite the checkerboard texture file, which is probably not what you intended.

**Comment by bheisler** [comment]
Thanks for the pull request! This is something I should have added myself already. Unfortunately, it's not quite right.

**Comment by JonathanMH** [comment]
Haha, true! Sorry, I just look at my git client and saw the output show up, I was just like: Oh yeah, that worked!

Sorry about that, I read your post and it's very very cool work!

**Comment by bheisler** [comment]
Thanks again for the contribution and the kind words, but I've added my own instructions.

**Pull request closed without merging**